docs/guides/advanced/data/4844_data.md

```diff
@@ -0,0 +1,185 @@
+# Understanding zkSync's Use of 4844 Blob Transactions
+
+This article explains how zkSync is utilizing a new type of transaction called 4844 blob transactions to transfer its
+public data (pubdata) to Ethereum Layer 1 (L1).
+
+We will skip the technical details of 4844 blobs and focus on how zkSync specifically implements them.
+
+## Before 4844: The Era of Calldata
+
+Originally, before the introduction of 4844 blobs, the system worked differently. In this period, known as the pre-4844
+era, zkSync would collect all its pubdata, which could be up to 120kb. This data was then sent as 'calldata' of the
+commitBatches transaction.
+
+In these transactions, the contract would calculate a keccak hash of the data and use this hash in the proving process.
+More precisely, this hash was stored as part of the block commitment and later used in the `proveBatches` function.
+
+## The Change with 4844 Blob Transactions
+
+In January 2024, the introduction of 4844 blobs brought several significant changes:
+
+- Blob transactions have their own fee market. This means the cost of using blobs can be different from the cost of
+  using gas.
+- Users must pay for a whole blob, which is 128kb, regardless of how much data they actually use. This differs from
+  calldata, where payment was based on the amount of data sent.
+- The contents of the blobs are not directly accessible. Instead, they contain unique KZG commitments and precompiles.
+
+**In the first version, we will be using up to 2 blobs per batch - so 256kb of pubdata**
+
+In this article, we will mainly discuss how we adapted our system, including our proving system, to work with these KZG
+commitments.
+
+**Why did Ethereum introduce these commitments instead of continuing with the previous hash-based system?**
```

Comment:
the previous hash-based system is the zkSync way, not Ethereum. Maybe replace with "raw data".